recipes/fontconfig/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = "2.11.1" %}
+
+package:
+  name: fontconfig
+  version: {{ version }}
+
+source:
+  fn: fontconfig-{{ version }}.tar.bz2
+  url: http://www.freedesktop.org/software/fontconfig/release/fontconfig-{{ version }}.tar.bz2
+  md5: 824d000eb737af6e16c826dd3b2d6c90
+  patches:
+    # Fixes a bug seen on Linux ( https://bugs.freedesktop.org/show_bug.cgi?id=77252 ).
+    # The patch is already present in upstream. See link below.
+    # ( https://cgit.freedesktop.org/fontconfig/commit/?id=f44157c809d280e2a0ce87fb078fc4b278d24a67 ).
+    - fc-cache.patch  # [linux]
+    # This patch won't be submitted upstream. It's use here is to ensure the prefix used
+    # to refer to the `fonts.conf` file.
+    - fcf.patch       # [linux]
```

Comment:
@ccordoba12, do you have any idea if these should be needed on Mac? I would think so, but am not sure.